### PR TITLE
add abs and sign functions

### PR DESCRIPTION
**Pull request by earney**
This is a new PR for adding Abs and Sign functions.

### CODE REVIEW

File: core/src/executor/evaluate/function.rs [hunk 1]
```diff
@@ -215,6 +215,29 @@ pub fn substr(
...
+        Value::I64(v) => Ok(Value::I64(v.abs())),
+        Value::Decimal(v) => Ok(Value::Decimal(v.abs())),
+        Value::F64(v) => Ok(Value::F64(v.abs())),
+        Value::Null => {
+            return Ok(Value::Null);
+        }
+        _ => {
+            return Err(EvaluateError::FunctionRequiresFloatValue(name).into());
+        }
+    }
```
Comment (ever0de) [inline core/src/executor/evaluate/function.rs:230]:
```suggestion
        Value::Null => Ok(Value::Null),
        _ => Err(EvaluateError::FunctionRequiresFloatValue(name).into())
    }
```
The rest of the value passed in the `match` is used as the return value, so I think you can delete `return` too

**Comment by panarch** [comment]
I can squash and merge to bundle the commits in the PR so you don't need to hesitate adding new commits as many as you want.

**Comment by panarch** [comment]
You can run these commands below before the push. It would be useful.
`cargo fmt`, `cargo clippy --all-targets` and `cargo test` in the root of the repo.

**Comment by earney** [comment]
Sorry about that..  I think I have it fixed now.

**Pull request closed without merging**